src/common/prop.c

```diff
@@ -866,3 +869,153 @@ daos_prop_entry_cmp_acl(struct daos_prop_entry *entry1,
 
 	return 0;
 }
+
+static int
+parse_entry(char *str, struct daos_prop_entry *entry)
+{
+	char	*name, *val, *end_token;
+	int	rc = 0;
+
+	/** get prop_name */
+	name = strtok_r(str, ":", &end_token);
+	/** get prop value */
+	val = strtok_r(NULL, ";", &end_token);
+
+	if (strcmp(name, "label") == 0) {
+		entry->dpe_type = DAOS_PROP_CO_LABEL;
+		rc = daos_prop_entry_set_str(entry, val, strlen(val));
+	} else if (strcmp(name, "cksum") == 0) {
+		entry->dpe_type = DAOS_PROP_CO_CSUM;
+		rc = daos_str2csumcontprop(val);
+		if (rc < 0)
+			return rc;
+		entry->dpe_val = rc;
+		rc = 0;
+	} else if (strcmp(name, "cksum_size") == 0) {
```

Comment:
It may be a good idea to define these names in a header file, in case the control plane needs to use them for consistency.

Reply:
makes sense.